hack/update-generated-agent-proto.sh

```diff
@@ -3,4 +3,5 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
-protoc --proto_path=protocols/grpc --go_out=plugins=grpc:protocols/grpc protocols/grpc/hyperstart.proto protocols/grpc/oci.proto
+go get -u github.com/gogo/protobuf/gogoproto/
```

Comment:
Please do not try to update the binary, which can be time consuming for us.

Reply:
@bergwolf Done.